lib/server.js

```diff
@@ -113,6 +208,23 @@ server.get('/tasks/:task/items/:item', function(req, res, next) {
     .catch(next);
 });
 
+/**
+ * Put an item in a task
+ * @name /tasks/:task/items/:item
+ * @param {Object} params - what the keys in the url mean
+ * @param {String} params.task - the task id
+ * @param {String} params.item - the item id
+ * @param {Object} body - the body of the request
+ * @param {('unlocked' | 'locked')} [body.lock] - lock
+ * @param {[Lon, Lat]} [body.pin] - a 2d geometry point to represent the feature
+ * @param {('open' | 'fixed' | 'noterror')} [body.status] - the status of current item
+ * @param {FeatureCollection} [body.featureCollection] - featureCollection
+ * @param {String} [body.instructions] - instructions on how to work on the task
+ * @example
```

Comment:
It would be nice to have an example of each of the request types and comments that explain why you'd do one rather than the other.